src/main/java/com/minecolonies/coremod/colony/permissions/Permissions.java

```diff
@@ -136,7 +136,7 @@ public Permissions(@NotNull final Colony colony)
         this.setPermission(Rank.OWNER, Action.TELEPORT_TO_COLONY);
         this.setPermission(Rank.OWNER, Action.RECEIVE_MESSAGES_FAR_AWAY);
         this.setPermission(Rank.OWNER, Action.CAN_KEEP_COLONY_ACTIVE_WHILE_AWAY);
-
+        this.setPermission(Rank.OWNER, Action.RALLY_GUARDS);
```

Comment:
I think we can add Officer's here too and you need to bump up the permissions version ontop on changes for backwards compat stuff

Reply:
Changed in 6a08fe3.